<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>880</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#listWidget::Item
{
	width:450px;
	height:90px;
}
#listWidget::Item:hover
{
	background:rgb(242,242,242);
}
#listWidget::Item:selected
{
	background:rgb(235,235,235);
	
	
}
#listWidget:focus
{
	outline:0px;
}
#listWidget
{
	border:none;
	border-top:1px solid rgb(240, 240, 240);
	border-bottom:1px solid rgb(240, 240, 240);

}

QPushButton
{border:none;
color:rgb(130,130,130);
	font: 12pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton:hover{color:rgb(0,0,0)
}
QPushButton:focus{color:rgb(139, 133, 72);
border-bottom:1px solid rgb(139, 133, 72);
}
QScrollBar:vertical
{
    width:15px;
	
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:9px;   
    padding-bottom:9px;
}
QScrollBar::handle:vertical
{
    width:15px;
    background:rgba(0,0,0,25%);
    border-radius:6px;   
    min-height:5;
}
QScrollBar::handle:vertical:hover
{
    width:15px;
    background:rgba(0,0,0,50%);  
    border-radius:6px;
    min-height:20;
}
QScrollBar::add-line:vertical {
      border: none;
      background: none;
}
QScrollBar::sub-line:vertical {
      border: none;
      background: none;
}</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>279</y>
     <width>453</width>
     <height>537</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="menuBtn">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#menuBtn{
	image: url(:/resources/img/menu.png);
}
#menuBtn:hover{
	background-color: rgb(242, 242, 242);
}
#menuBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addFriendBtn">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#addFriendBtn{
	
	image: url(:/resources/img/append.png);
}
#addFriendBtn:hover{
	background-color: rgb(242, 242, 242);
}
#addFriendBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="txDocBtn">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#txDocBtn{
	
	
	image: url(:/resources/img/wendang.png);
}
#txDocBtn:hover{
	background-color: rgb(242, 242, 242);
}
#txDocBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="guanliqiBtn">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#guanliqiBtn{
	
	image: url(:/resources/img/mang.png);
}
#guanliqiBtn:hover{
	background-color: rgb(242, 242, 242);
}
#guanliqiBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="weiyunBtn">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#weiyunBtn{
	
	
	image: url(:/resources/img/yun.png);
}
#weiyunBtn:hover{
	background-color: rgb(242, 242, 242);
}
#weiyunBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gameBtn2">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gameBtn2{
	
	image: url(:/resources/img/game2.png);
}
#gameBtn2:hover{
	background-color: rgb(242, 242, 242);
}
#gameBtn2:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gameBtn">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>821</y>
     <width>58</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gameBtn{
	
	image: url(:/resources/img/game1.png);
}
#gameBtn:hover{
	background-color: rgb(242, 242, 242);
}
#gameBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>453</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MessageBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>237</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pindaoBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>237</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>频道</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kongjianBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>237</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>空间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lianxirenBtn">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>237</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>联系人</string>
   </property>
  </widget>
  <widget class="QLabel" name="background3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>453</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>453</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:30px;
font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);
background-color: rgb(217, 217, 195);
border : none;</string>
   </property>
   <property name="placeholderText">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="profileLab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="singleLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>453</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/resources/img/bg1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/resources/img/QQ4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>12</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">



#pushButton{
image: url(:/resources/img/email.png);
}

#pushButton:hover{
	background-color: rgb(226, 226, 214);
}
#pushButton:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>12</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


#pushButton_2{
image: url(:/resources/img/kongjian.png);
}

#pushButton_2:hover{
	background-color: rgb(226, 226, 214);
}
#pushButton_2:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>12</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">


#pushButton_3{image: url(:/resources/img/zhuangban.png);}
#pushButton_3:hover{
	background-color: rgb(226, 226, 214);
}
#pushButton_3:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>12</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

#pushButton_4{image: url(:/resources/img/xiaoshijie.png);}
#pushButton_4:hover{
	background-color: rgb(226, 226, 214);
}
#pushButton_4:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>12</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

#pushButton_5{image: url(:/resources/img/3point.png);}
#pushButton_5:hover{
	background-color: rgb(226, 226, 214);
}
#pushButton_5:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>12</y>
     <width>41</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#minBtn{image: url(:/resources/img/min2.png);}
#minBtn:hover{
	background-color: rgb(226, 226, 214);
}
#minBtn:focus{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>12</y>
     <width>41</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#closeBtn{image: url(:/resources/img/close2.png);}
#closeBtn:hover
{
	
	background-color: rgb(255, 84, 57);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>176</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/resources/img/sousuo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vipLab">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background1</zorder>
  <zorder>background3</zorder>
  <zorder>background2</zorder>
  <zorder>listWidget</zorder>
  <zorder>menuBtn</zorder>
  <zorder>addFriendBtn</zorder>
  <zorder>txDocBtn</zorder>
  <zorder>guanliqiBtn</zorder>
  <zorder>weiyunBtn</zorder>
  <zorder>gameBtn2</zorder>
  <zorder>gameBtn</zorder>
  <zorder>MessageBtn</zorder>
  <zorder>pindaoBtn</zorder>
  <zorder>kongjianBtn</zorder>
  <zorder>lianxirenBtn</zorder>
  <zorder>lineEdit</zorder>
  <zorder>profileLab</zorder>
  <zorder>singleLab</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>minBtn</zorder>
  <zorder>closeBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>vipLab</zorder>
  <zorder>nameLab</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>Home</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
